secrets/gcpssm/gcp_ssm.go

```diff
@@ -0,0 +1,168 @@
+package gcpssm
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/0xPolygon/polygon-edge/secrets"
+	"github.com/hashicorp/go-hclog"
+	"os"
+
+	sm "cloud.google.com/go/secretmanager/apiv1"
+	smpb "google.golang.org/genproto/googleapis/cloud/secretmanager/v1"
+)
+
+type GcpSsmManager struct {
```

Comment:
Nitpick: What do you think about naming this `GCPSSMManager`?

Reply:
Renamed b95eb9d2fcd7a213bc19ef0ab427cde103615642